kombu/transport/SQS.py

```diff
@@ -425,10 +425,17 @@ def _create_queue(self, queue_name, attributes):
             self.transport_options.get('sqs-creation-attributes') or {},
         )
 
-        return self.sqs(queue=queue_name).create_queue(
-            QueueName=queue_name,
-            Attributes=attributes,
-        )
+        queue_tags = self.transport_options.get('sqs-queue-tags')
```

Comment:
one knitpick that i see is that the naming scheme is `lower-kebab-case`. the rest of the `transport_options` are `lower_snake_case`.

except one: `sqs-creation-attributes` which was introduced in https://github.com/celery/kombu/pull/957 and that was in end of 2018.

i am in favour of _python/PEP_ default `lower_snake_case`, but probably should keep it as-is for 2018-consistency :(